### PR TITLE
Update DG functions and other minor UG fixes

### PR DESCRIPTION
**Pull request by Jajared**

### CODE REVIEW

File: docs/diagrams/DeleteTeamSequence.puml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @startuml
 !include style.puml
 skinparam ArrowFontStyle plain
-!define input "/delete_team module/CS2103T tutorial/T09 name/Team 1"
+!define input "/delete_team module/CS2103T tutorial/T09 tutorial/Team 1"
```
Comment (Zack-Tay) [inline docs/diagrams/DeleteTeamSequence.puml:4]: Hi @Jajared, is there a typo here? it should be team instead?

File: docs/diagrams/DeleteTeamSequence.puml [hunk 2]
```diff
@@ -29,7 +29,7 @@ activate DeleteTeamCommandParser
 DeleteTeamCommandParser --> AddressBookParser
 deactivate DeleteTeamCommandParser
 
-AddressBookParser -> DeleteTeamCommandParser : parse("module/CS2103T tutorial/T09 name/Team 1")
+AddressBookParser -> DeleteTeamCommandParser : parse("module/CS2103T tutorial/T09 tutorial/Team 1")
```
Comment (Zack-Tay) [inline docs/diagrams/DeleteTeamSequence.puml:32]: Likewise here as well.

File: docs/diagrams/DeleteTeamSequence.puml (hunk 1 above)
Comment (Jajared) [reply docs/diagrams/DeleteTeamSequence.puml:4]: Yep should team my bad

**Review by Zack-Tay — APPROVED** [review]
LGTM

**Pull request merged**